modules/Blockchain/Ethereum/index.js

```diff
@@ -34,6 +37,8 @@ class Ethereum {
         };
         Object.assign(this.config, config.blockchain);
 
+        this.config[blockchain_id].erc725Identity = this._loadIdentityFromFile();
```

Comment:
Take a look at the PR for event listeners, only the implementation specific section will be added to the config. So this should be `this.config.erc725identity = this._loadIdentityFromFile()`